addons/editor-compact/userstyle.css

```
@@ -0,0 +1,492 @@
+.scratchCategoryItemIcon {
+  width: 0.75rem;
+  height: 0.75rem;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 0.6rem 0.6rem;
+}
+.scratchCategoryItemIcon {
+  background-size: 0.75rem 0.75rem;
+}
+[class*="gui_tab-panel"]:not([iscolumns="true"]) .blocklyToolboxDiv {
+  height: calc(100% - 2rem) !important;
+  scrollbar-width: none;
+}
+[class*="gui_tab-panel"]:not([iscolumns="true"]) .blocklyToolboxDiv::-webkit-scrollbar {
+  display: none;
+}
+[class*="gui_tab-panel"]:not([iscolumns="true"]) [class*="gui_extension-button-container_"] {
+  height: 2rem;
+}
+[iscolumns="true"] [class*="gui_extension-button-container_"] {
+  width: calc(210px - 2px - 1rem);
+}
+
+/* Assets panel */
+[class*="selector_wrapper"] {
+  width: 100px;
+}
+[class*="selector_list-item"] {
+  margin: 0.25rem auto;
+}
+[class*="selector_list-area"] {
+  overflow-y: auto;
+}
+[class*="selector_list-area"] > div:first-child {
+  margin-top: 0.25rem;
+}
+[class*="selector_list-area"] [class*="sprite-selector-item_delete-button"] {
+  top: -0.5rem;
+}
+[class*="selector_list-area"] img[class*="sprite-selector-item_sprite-image"] {
+  max-width: 52px;
+  max-height: 44px;
+}
+#react-tabs-5 [class*="selector_list-area"] > div > div {
+  height: 2rem;
+}
+#react-tabs-5 [class*="sprite-selector-item_sprite-image-outer"] {
+  display: none;
+}
+[class*="selector_list-area"] [class*="sprite-selector-item_sprite-name"] {
+  margin: 0;
+}
+#react-tabs-5 [class*="selector_list-area"] [class*="sprite-selector-item_sprite-name"] {
+  margin-top: 0.15rem;
+}
+[class*="selector_list-area"] [class*="sprite-selector-item_sprite-info"] {
+  padding-top: 0;
+  padding-bottom: 0.125rem;
+}
+#react-tabs-5 [class*="selector_list-area"] [class*="sprite-selector-item_sprite-info"] {
+  padding-left: 1rem;
+  text-align: left;
+}
+
+/* Asset editor */
+[class*="asset-panel_detail-area"] > div {
+  padding: 0.5rem;
+}
+[class*="paint-editor_row_"] input {
+  font-size: 10px;
+}
+[class*="asset-panel_detail-area"] [class*="labeled-icon-button_mod-edit-field"],
+[class*="sound-editor_tool-button_"],
+[class*="sound-editor_row-reverse_"] > [class*="icon-button_container_"] {
+  min-width: 0;
+  margin-right: 2px;
+  padding: 0;
+  border: 1px solid var(--editorDarkMode-border, hsla(0, 0%, 0%, 0.15));
+  border-radius: 0.25rem;
+}
+[class*="asset-panel_detail-area"] [class*="labeled-icon-button_edit-field-icon"],
+[class*="sound-editor_button_"] img,
+[class*="sound-editor_tool-button"] img {
+  width: 20px;
+  height: 20px;
+}
+[class*="asset-panel_detail-area"] [class*="label_input-label"],
+[class*="asset-panel_detail-area"] [class*="labeled-icon-button_edit-field-title"],
+[class*="icon-button_title_"] {
+  display: none;
+}
+[class*="asset-panel_detail-area"] [class*="button_button"],
+[class*="sound-editor_button_"] {
+  padding: calc(0.125rem - 1px);
+}
+[class*="asset-panel_detail-area"] [class*="dashed-border"] {
+  border-right: none;
```

This doesn't work in RTL.
```suggestion
  border: none;
```